backend/src/main/scala/com/softwaremill/bootzooka/http/EndpointsToRoutes.scala

```diff
@@ -61,11 +69,11 @@ class EndpointsToRoutes(http: Http, apiContextPath: String) {
 
   /** Interprets the given endpoint descriptions as docs, and returns http4s routes which expose the documentation using Swagger.
     */
-  def toDocsRoutes(es: ServerEndpoints): HttpRoutes[Task] = {
+  def toDocsRoutes(es: ServerEndpoints): HttpRoutes[IO] = {
     val openapi = OpenAPIDocsInterpreter()
       .serverEndpointsToOpenAPI(es.toList, "Bootzooka", "1.0")
       .servers(List(Server(s"$apiContextPath", None)))
-    val yaml = openapi.toYaml
-    new SwaggerHttp4s(yaml).routes[Task]
+    val swaggerRoutes: List[ServerEndpoint[_, _, _, Any, IO]] = SwaggerUI[IO](openapi.toYaml)
```

Comment:
tapir needs updating to 0.19-final :)

Reply:
also maybe we can use the bundle: https://tapir.softwaremill.com/en/latest/docs/openapi.html#using-swagger